test/integration/script-loader/test/index.test.js

```diff
@@ -159,7 +159,7 @@ describe('Next.js Script - Primary Strategies', () => {
       let documentBIScripts = await browser.elementsByCss(
         '[src$="scriptBeforeInteractive"]'
       )
-      expect(documentBIScripts.length).toBe(1)
+      expect(documentBIScripts.length).toBe(2)
```

Comment:
Could we also add some tests for this in the isolated next-script suite here https://github.com/vercel/next.js/blob/92915e47f7122b332f7921f1e129a7bdca493e1f/test/e2e/next-script/index.test.ts#L7

Reply:
Done 🙏